src/main/java/file/methods/FileHandler.java

```diff
@@ -0,0 +1,83 @@
+package file.methods;
+
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+import tasks.Task;
+import tasks.Todo;
+import tasks.Event;
+import tasks.Deadline;
+
+public class FileHandler {
+
+    public static void setUpFile(ArrayList<Task> actions) {
```

Comment:
This function is also quite long, maybe see how you can refactor some things inside it.